src/main/java/program/Main.java

```diff
@@ -1,10 +1,47 @@
 package program;
 
+import db.PostgresqlConnectionProvider;
+import model.ThingBean;
+import repository.ThingRepository;
+import view.MainFrame;
 import view.planning.EventPlanningWindow;
 
+import javax.swing.*;
+import java.awt.*;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
 public class Main {
+
     public static void main(String[] args) {
         EventPlanningWindow eventPlanningWindow = new EventPlanningWindow();
         eventPlanningWindow.setVisible(true);
+
```

Comment:
Comments for trial and error are ok locally, but you shouldn't commit them for review.